configuration-service/deploy/service.yaml

```diff
@@ -60,7 +60,7 @@ spec:
             memory: "256Mi"
             cpu: "500m"
           limits:
-            memory: "256Mi"
+            memory: "1024Mi"
```

Comment:
The higher limit is only needed for debugging, right?

Reply:
yes - it's not directly related to the issue but with the previous setting i got an OOM error when trying to attach the debugger